projects/demo/src/modules/icons/icons.template.html

```diff
@@ -45,17 +45,27 @@
                 let-copyPath="copyPath"
                 let-entry="entry"
                 let-icon="icon"
+                let-type="type"
             >
                 <div
                     class="icon-place"
                     (waIntersectionObservee)="entry.isIntersecting = entry.isIntersecting || onIntersection($event)"
                 >
-                    <tui-svg
-                        *ngIf="entry.isIntersecting"
-                        class="icon"
-                        [src]="icon"
-                        (click)="copyPath(icon)"
-                    />
+                    <ng-container *ngIf="entry.isIntersecting">
```

Comment:
Would it be possible to use tuiIconButton with optional img inside instead, to keep it semantic/accessible?